tests/foreman/cli/test_repository.py

```diff
@@ -139,6 +142,24 @@ def gpg_key(module_org):
     return make_content_credential({'organization-id': module_org.id})
 
 
+@pytest.fixture(scope='class')
+def host_setup(module_org):
+    with manifests.clone(name='golden_ticket') as manifest:
+        upload_manifest(module_org.id, manifest.content)
+    new_product = make_product({'organization-id': module_org.id})
+    new_repo = make_repository({'product-id': new_product['id']})
+    Repository.synchronize({'id': new_repo['id']})
+    new_ak = make_activation_key(
```

Comment:
let's try to use the new helpers pattern
```suggestion
    new_product = class_target_sat.cli_factory.make_product({'organization-id': module_org.id})
    new_repo = class_target_sat.cli_factory.make_repository({'product-id': new_product['id']})
    Repository.synchronize({'id': new_repo['id']})
    new_ak = class_target_sat.cli_factory.make_activation_key(
```